File: est datos/c#/burbuja.cs
using System;
using System.ComponentModel.Design;
class Program{
    static void Main(string[] args)
    {
        Console.WriteLine("Ingrese el tamaño del arreglo:");
        int tam = int.Parse(Console.ReadLine());

        int[] arr = new int[tam];

        for (int i = 0; i < tam; i++) {
            arr[i] = new Random().Next(1, 100);
        }

        Console.WriteLine("Arreglo original: " + string.Join(", ", arr));

        bool swapped = true;

        do {
            swapped = false;
            for (int i = 0; i < tam - 1; i++) {
                if (arr[i] > arr[i + 1]) {
                    int temp = arr[i];
                    arr[i] = arr[i + 1];
                    arr[i + 1] = temp;
                    swapped = true;
                }
            }


        } while (swapped == true);

        Console.WriteLine("Arreglo ordenado: " + string.Join(", ", arr));

    }
}

File: est datos/python/burbuja.py
import random

tam = int(input("Ingrese el tamaño del arreglo: "))

arreglo = []

for i in range(tam):
    arreglo.append(random.randint(1, 100))

print("Arreglo original:", arreglo)

swapped = True
while swapped:
    swapped = False
    for i in range(1, len(arreglo)):
        if arreglo[i - 1] > arreglo[i]:
            arreglo[i - 1], arreglo[i] = arreglo[i], arreglo[i - 1]
            swapped = True

print("Arreglo ordenado:", arreglo)